cmake/SSGCommon.cmake

```diff
@@ -303,6 +303,19 @@ macro(ssg_build_cpe_oval_unlinked PRODUCT)
     )
 endmacro()
 
+macro(ssg_build_manifest PRODUCT)
+    add_custom_command(
+        OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/manifest.json"
+        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${SSG_BUILD_SCRIPTS}/generate_manifest.py" --output "${CMAKE_CURRENT_BINARY_DIR}/manifest.json" --build-root "${CMAKE_CURRENT_BINARY_DIR}"
+	DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/cpe-oval-unlinked.xml"
```

Comment:
Why is the dependency cpe-oval-unlinked?

Reply:
I mean: shouldn't this cmake command depend on a different target?

Reply:
Because the generator uses compiled artifacts. While most of artifacts are compiled at the beginning of the build, the per-rule OVALs are made available only after that step that is marked as a dependency.
I will add a comment to clarify that, it is indeed confusing.